draft-wang-ppm-dap-taskprov.md

```diff
@@ -68,31 +72,165 @@ many deployments.
 >
 > - Which of the sub-protocols are changed: upload, aggregate, collect?
 
+> TODO: talk about client logging task configuration and checking against
+parameters like task_expiration.
+
 
 # Conventions and Definitions
 
 {::boilerplate bcp14-tagged}
 
 
-# The "task_prov" Extension
+# The "task-prov" Extension
 
 > NOTE(cjpatton) It will be useful to think of a good name for this. Something
-> succinct and descriptive is ideal. `task_prov` is fine, but we can maybe do
+> succinct and descriptive is ideal. `task-prov` is fine, but we can maybe do
 > better.
 
 A new extension is defined:
 
 ~~~
 enum {
-   task_prov(0xff00), (65535)
+  task-prov(0xff00),
+  (65535)
 } ExtensionType;
 ~~~
+> NOTE(shan) TLS uses underscore `_` to define codepoints, which is different
+> from DAP's convension of using hyphen `-`, which one to follow?
```

Comment:
Good catch. I think underscore is better, but consistency with DAP is more important. I filed https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/issues/314 to figure this out.